sklearn/linear_model/tests/test_base.py

```diff
@@ -165,6 +211,32 @@ def test_linear_regression_sparse(random_state=0):
         assert_array_almost_equal(ols.predict(X) - y.ravel(), 0)
 
 
+@pytest.mark.parametrize(
+    'normalize, n_warnings, warning',
+    [(True, 1, FutureWarning),
+     (False, 1, FutureWarning),
+     ("deprecated", 0, None)]
+)
+# FIXME remove test in 1.4.0
+def test_assure_warning_when_normalize(normalize, n_warnings, warning):
+    # check that we issue a FutureWarning when normalize was set
+    rng = check_random_state(0)
+    n_samples = 200
+    n_features = 2
+    X = rng.randn(n_samples, n_features)
+    X[X < 0.1] = 0.
+    y = rng.rand(n_samples)
+
+    model = LinearRegression(normalize=normalize)
+    with pytest.warns(warning) as record:
+        model.fit(X, y)
+    assert len(record) == n_warnings
+    if n_warnings:
+        assert "'normalize' was deprecated" in str(record[0].message)
+
+
+# FIXME: 'normalize' to be removed in 1.2.0 in LinearRegression
```

Comment:
```suggestion
# FIXME: 'normalize' to be removed in 1.2 in LinearRegression
```

Reply:
Do we really need the minor version component in deprecation messages. We do not usually deprecate stuff in bugfix releases, so I don't think it's necessary.